From 14606b62bd76c310ab360003b0844ca12d463fb4 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 13 Dec 2020 19:21:02 -0500
Subject: [PATCH] clean gapps set

---
 products/gms.mk | 32 +-------------------------------
 1 file changed, 1 insertion(+), 31 deletions(-)

diff --git a/products/gms.mk b/products/gms.mk
index 3d7b5dc..7586980 100644
--- a/products/gms.mk
+++ b/products/gms.mk
@@ -16,22 +16,16 @@ GMS_PRODUCT_PACKAGES += \
     AndroidPlatformServices \
     ConfigUpdater \
     GoogleExtShared \
-    GoogleFeedback \
     GoogleLocationHistory \
     GoogleOneTimeInitializer \
-    GooglePackageInstaller \
-    GooglePartnerSetup \
     GooglePrintRecommendationService \
     GoogleRestore \
     GoogleServicesFramework \
     GoogleCalendarSyncAdapter \
     GoogleContactsSyncAdapter \
-    GoogleTTS \
     GmsCore \
     Phonesky \
-    SetupWizard \
-    WebViewGoogle \
-    Wellbeing
+    WebViewGoogle
 
 # GMS common RRO packages
 GMS_PRODUCT_PACKAGES += GmsConfigOverlayCommon GmsConfigOverlayGSA
@@ -58,39 +52,15 @@ PRODUCT_PACKAGE_OVERLAYS += $(ANDROID_PARTNER_GMS_HOME)/overlay/gms_overlay
 
 # GMS mandatory application packages
 GMS_PRODUCT_PACKAGES += \
-    Chrome \
-    Drive \
-    Gmail2 \
-    Duo \
-    Maps \
-    YTMusic \
-    Photos \
     Velvet \
-    Videos \
-    YouTube \
     AssistantShell
 
 # GMS comms suite
 $(call inherit-product, $(ANDROID_PARTNER_GMS_HOME)/products/google_comms_suite.mk)
 
-# GMS optional application packages
-GMS_PRODUCT_PACKAGES += \
-    CalendarGoogle \
-    LatinImeGoogle
-
-GMS_PRODUCT_PACKAGES += \
-    FilesGoogle
-
-# GMS sample application packages
-GMS_PRODUCT_PACKAGES += \
-    SearchLauncherQuickStep
-
 # Add GMS package into system product packages
 PRODUCT_PACKAGES += $(GMS_PRODUCT_PACKAGES)
 
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
-    ro.boot.vendor.overlay.theme=com.android.internal.systemui.navbar.threebutton;com.android.theme.icon.squircle
-
 # Add acsa property for CarrierServices
 PRODUCT_PRODUCT_PROPERTIES += \
     ro.com.google.acsa=true
-- 
2.17.1

