From efe1149ac6c331dd042bc58f523e1905623b5b8d Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 19 Dec 2020 05:34:34 -0500
Subject: [PATCH 11/17] rework securize tweak

---
 rw-system.sh | 71 ++++++++++++++++++++++++++--------------------------
 1 file changed, 36 insertions(+), 35 deletions(-)

diff --git a/rw-system.sh b/rw-system.sh
index f0e127c..1864b2b 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -582,42 +582,41 @@ if getprop ro.vendor.build.fingerprint | grep -qiE '^samsung/' && [ "$vndk" -ge
     fi
 fi
 
-if [ -f /system/phh/secure ];then
-    copyprop() {
-        p="$(getprop "$2")"
-        if [ "$p" ]; then
-            resetprop "$1" "$(getprop "$2")"
-        fi
-    }
+copyprop() {
+    p="$(getprop "$2")"
+    if [ "$p" ]; then
+        resetprop "$1" "$(getprop "$2")"
+    fi
+}
 
-    setprop ctl.stop console
-    dmesg -n 1
-    copyprop ro.build.device ro.vendor.build.device
-    copyprop ro.system.build.fingerprint ro.vendor.build.fingerprint
-    copyprop ro.bootimage.build.fingerprint ro.vendor.build.fingerprint
-    copyprop ro.build.fingerprint ro.vendor.build.fingerprint
-    copyprop ro.build.device ro.vendor.product.device
-    copyprop ro.product.system.device ro.vendor.product.device
-    copyprop ro.product.device ro.vendor.product.device
-    copyprop ro.product.system.device ro.product.vendor.device
-    copyprop ro.product.device ro.product.vendor.device
-    copyprop ro.product.system.name ro.vendor.product.name
-    copyprop ro.product.name ro.vendor.product.name
-    copyprop ro.product.system.name ro.product.vendor.device
-    copyprop ro.product.name ro.product.vendor.device
-    copyprop ro.system.product.brand ro.vendor.product.brand
-    copyprop ro.product.brand ro.vendor.product.brand
-    copyprop ro.product.system.model ro.vendor.product.model
-    copyprop ro.product.model ro.vendor.product.model
-    copyprop ro.product.system.model ro.product.vendor.model
-    copyprop ro.product.model ro.product.vendor.model
-    copyprop ro.build.product ro.vendor.product.model
-    copyprop ro.build.product ro.product.vendor.model
-    copyprop ro.system.product.manufacturer ro.vendor.product.manufacturer
-    copyprop ro.product.manufacturer ro.vendor.product.manufacturer
-    copyprop ro.system.product.manufacturer ro.product.vendor.manufacturer
-    copyprop ro.product.manufacturer ro.product.vendor.manufacturer
-    copyprop build.version.security_patch ro.build.version.security_patch
+copyprop ro.build.device ro.vendor.build.device
+copyprop ro.system.build.fingerprint ro.vendor.build.fingerprint
+copyprop ro.bootimage.build.fingerprint ro.vendor.build.fingerprint
+copyprop ro.build.fingerprint ro.vendor.build.fingerprint
+copyprop ro.build.device ro.vendor.product.device
+copyprop ro.product.system.device ro.vendor.product.device
+copyprop ro.product.device ro.vendor.product.device
+copyprop ro.product.system.device ro.product.vendor.device
+copyprop ro.product.device ro.product.vendor.device
+copyprop ro.product.system.name ro.vendor.product.name
+copyprop ro.product.name ro.vendor.product.name
+copyprop ro.product.system.name ro.product.vendor.device
+copyprop ro.product.name ro.product.vendor.device
+copyprop ro.system.product.brand ro.vendor.product.brand
+copyprop ro.product.brand ro.vendor.product.brand
+copyprop ro.product.system.model ro.vendor.product.model
+copyprop ro.product.model ro.vendor.product.model
+copyprop ro.product.system.model ro.product.vendor.model
+copyprop ro.product.model ro.product.vendor.model
+copyprop ro.build.product ro.vendor.product.model
+copyprop ro.build.product ro.product.vendor.model
+copyprop ro.system.product.manufacturer ro.vendor.product.manufacturer
+copyprop ro.product.manufacturer ro.vendor.product.manufacturer
+copyprop ro.system.product.manufacturer ro.product.vendor.manufacturer
+copyprop ro.product.manufacturer ro.product.vendor.manufacturer
+copyprop build.version.security_patch ro.build.version.security_patch
+
+if [ -f /system/phh/secure ];then
     (getprop ro.vendor.build.security_patch; getprop ro.keymaster.xxx.security_patch) |sort |tail -n 1 |while read v;do
         [ -n "$v" ] && resetprop ro.build.version.security_patch "$v"
     done
@@ -636,6 +635,8 @@ if [ -f /system/phh/secure ];then
 
     resetprop ro.adb.secure 1
     setprop ctl.restart adbd
+    setprop ctl.stop console
+    dmesg -n 1
 fi
 
 for abi in "" 64;do
-- 
2.17.1

