From f0bc8a614e42e27978b7bb22e531c77625886f2a Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Thu, 24 Dec 2020 07:14:51 -0500
Subject: [PATCH 14/37] tweaks for Xiaomi Redmi Note 9 Pro 'joyeuse'

---
 rw-system.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rw-system.sh b/rw-system.sh
index b837b3d..0d7e233 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -404,7 +404,7 @@ if getprop ro.vendor.build.fingerprint | grep -iq \
     -e motorola/hannah -e motorola/james -e motorola/pettyl -e xiaomi/cepheus \
     -e xiaomi/grus -e xiaomi/cereus -e xiaomi/cactus -e xiaomi/raphael -e xiaomi/davinci \
     -e xiaomi/ginkgo -e xiaomi/laurel_sprout -e xiaomi/andromeda \
-    -e redmi/curtana -e redmi/picasso \
+    -e redmi/curtana -e redmi/joyeuse -e redmi/picasso \
     -e bq/Aquaris_M10 ; then
     mount -o bind /mnt/phh/empty_dir /vendor/lib64/soundfx
     mount -o bind /mnt/phh/empty_dir /vendor/lib/soundfx
@@ -898,7 +898,7 @@ if getprop ro.vendor.build.fingerprint |grep -iq xiaomi/cepheus;then
     setprop ro.netflix.bsp_rev Q855-16947-1
 fi
 
-if getprop ro.vendor.build.fingerprint |grep -qi redmi/curtana;then
+if getprop ro.vendor.build.fingerprint |grep -qi -e redmi/curtana -e redmi/joyeuse; then
     setprop ro.netflix.bsp_rev Q6250-19132-1
 fi
 
-- 
2.30.2

