From b57c6b5c374b26a2fe99fc6b67f3afb88aa653db Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Thu, 24 Dec 2020 07:14:51 -0500
Subject: [PATCH 14/16] tweaks for Xiaomi Redmi Note 9 Pro 'joyeuse'

---
 rw-system.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/rw-system.sh b/rw-system.sh
index bc124c2..723eada 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -101,7 +101,7 @@ changeKeylayout() {
         -e xiaomi/nitrogen -e xiaomi/sakura -e xiaomi/andromeda \
         -e xiaomi/whyred -e xiaomi/tulip -e xiaomi/onc \
         -e redmi/curtana -e redmi/picasso -e Redmi/lancelot \
-        -e Redmi/galahad -e Redmi/merlin; then
+        -e Redmi/galahad -e Redmi/merlin -e redmi/joyeuse; then
         if [ ! -f /mnt/phh/keylayout/uinput-goodix.kl ]; then
           cp /system/phh/empty /mnt/phh/keylayout/uinput-goodix.kl
           chmod 0644 /mnt/phh/keylayout/uinput-goodix.kl
@@ -350,7 +350,7 @@ if getprop ro.vendor.build.fingerprint | grep -iq \
     -e motorola/hannah -e motorola/james -e motorola/pettyl -e xiaomi/cepheus \
     -e xiaomi/grus -e xiaomi/cereus -e xiaomi/cactus -e xiaomi/raphael -e xiaomi/davinci \
     -e xiaomi/ginkgo -e xiaomi/laurel_sprout -e xiaomi/andromeda \
-    -e redmi/curtana -e redmi/picasso ; then
+    -e redmi/curtana -e redmi/joyeuse -e redmi/picasso ; then
     mount -o bind /mnt/phh/empty_dir /vendor/lib64/soundfx
     mount -o bind /mnt/phh/empty_dir /vendor/lib/soundfx
     setprop  ro.audio.ignore_effects true
@@ -813,7 +813,7 @@ if getprop ro.vendor.build.fingerprint |grep -iq xiaomi/cepheus;then
     setprop ro.netflix.bsp_rev Q855-16947-1
 fi
 
-if getprop ro.vendor.build.fingerprint |grep -qi redmi/curtana;then
+if getprop ro.vendor.build.fingerprint |grep -qi -e redmi/curtana -e redmi/joyeuse; then
     setprop ro.netflix.bsp_rev Q6250-19132-1
 fi
 
-- 
2.17.1

