From 52d824c7f06480a54186127c1c290d39d8add34b Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Fri, 25 Dec 2020 02:29:20 -0500
Subject: [PATCH 15/17] removing FOSS Browser from GApps builds

---
 overrides/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/overrides/Android.mk b/overrides/Android.mk
index d50ee9d..251e40e 100644
--- a/overrides/Android.mk
+++ b/overrides/Android.mk
@@ -9,6 +9,7 @@ PACKAGES.$(LOCAL_MODULE).OVERRIDES := \
 	DuoGo \
 	Traceur \
 	Etar \
+	FOSSBrowser \
 	GalleryGo \
 	SimpleGallery \
 	Messaging \
@@ -62,6 +63,7 @@ PACKAGES.$(LOCAL_MODULE).OVERRIDES := \
 	SimpleGallery \
 	Messaging \
 	Etar \
+	FOSSBrowser \
 	GmsSampleIntegration \
 	GmsSampleIntegrationGo \
 	GmsSampleIntegrationGo2GB \
-- 
2.17.1

