From 2ea7773deb7363fac92ff3443e4aeb10e1630f2f Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 28 Mar 2021 11:24:18 -0400
Subject: [PATCH 24/38] add tweaks for Xiaomi Redmi Note 7 'lavender'

---
 rw-system.sh | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/rw-system.sh b/rw-system.sh
index 7a996da..1fd7ed0 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -542,12 +542,16 @@ fi
 
 if getprop ro.vendor.build.fingerprint | grep -iq -e xiaomi/daisy; then
     setprop debug.sf.latch_unsignaled 1
-    setprop debug.sf.enable_hwc_vds 1
     setprop ro.telephony.default_network 9,9
     setprop persist.vendor.radio.block_allow_data 0
     setprop telephony.lteOnGsmDevice 1
 fi
 
+if getprop ro.vendor.build.fingerprint | grep -iq -e xiaomi/lavender; then
+    setprop debug.sf.latch_unsignaled 1
+    setprop ro.netflix.bsp_rev Q660-13149-1
+fi
+
 if getprop ro.vendor.build.fingerprint | grep -iq -E -e 'huawei|honor' || getprop persist.sys.overlay.huawei | grep -iq -E -e 'true'; then
     p=/product/etc/nfc/libnfc_nxp_*_*.conf
     mount -o bind "$p" /system/etc/libnfc-nxp.conf ||
-- 
2.30.2

