From a8654e930b041f9a96ce177ab9e870c2d6657d21 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 16 May 2021 14:18:51 -0400
Subject: [PATCH 32/38] rw-system: set LTE by default for 'lavender'

---
 rw-system.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/rw-system.sh b/rw-system.sh
index c22ab51..5caddee 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -550,6 +550,9 @@ fi
 if getprop ro.vendor.build.fingerprint | grep -iq -e xiaomi/lavender; then
     setprop debug.sf.latch_unsignaled 1
     setprop ro.netflix.bsp_rev Q660-13149-1
+    setprop ro.telephony.default_network 9,9
+    setprop persist.vendor.radio.block_allow_data 0
+    setprop telephony.lteOnGsmDevice 1
 fi
 
 if getprop ro.vendor.build.fingerprint | grep -iq -E -e 'huawei|honor' || getprop persist.sys.overlay.huawei | grep -iq -E -e 'true'; then
-- 
2.30.2

