From aa775432b7802b560a637bb1d39ccb3bbbd28fb6 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 16 Sep 2020 06:46:11 -0400
Subject: [PATCH 4/5] moar generic

Change-Id: I95510d74f368a627c6d0b00584a35f36807bb9f0
---
 config/common.mk | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 296ee514..8e3b264f 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -97,12 +97,6 @@ PRODUCT_MINIMIZE_JAVA_DEBUG_INFO := true
 # Disable vendor restrictions
 PRODUCT_RESTRICT_VENDOR_FILES := false
 
-# Bootanimation
-TARGET_SCREEN_WIDTH ?= 1080
-TARGET_SCREEN_HEIGHT ?= 1920
-PRODUCT_PACKAGES += \
-    bootanimation.zip
-
 # AOSP packages
 PRODUCT_PACKAGES += \
     Terminal
-- 
2.17.1

